Standards/WCAG2AAA/ruleset.js

```diff
@@ -120,329 +137,457 @@ _global.HTMLCS_WCAG2AAA = {
+                landmark: 'labels-or-instructions',
                 priority: 1,
             },
             '3.3.3': {
                 name: 'Error Suggestion',
-                landmark: 'minimize-error-suggestions',
+                landmark: 'error-suggestion',
                 priority: 2,
             },
             '3.3.4': {
                 name: 'Error Prevention (Legal, Financial, Data)',
-                landmark: 'minimize-error-reversible',
+                landmark: 'error-prevention-legal-financial-data',
                 priority: 2,
             },
             '3.3.5': {
                 name: 'Help',
-                landmark: 'minimize-error-context-help',
+                landmark: 'help',
                 priority: 3,
             },
             '3.3.6': {
                 name: 'Error Prevention (All)',
-                landmark: 'minimize-error-reversible-all',
+                landmark: 'error-prevention-all',
                 priority: 3,
             },
             '4.1.1': {
                 name: 'Parsing',
-                landmark: 'ensure-compat-parses',
+                landmark: 'parsing',
                 priority: 1,
             },
             '4.1.2': {
                 name: 'Name, Role, Value',
-                landmark: 'ensure-compat-rsv',
+                landmark: 'name-role-value',
                 priority: 1,
             },
+            '4.1.3': {
+                name: 'Status Messages',
+                landmark: 'status-messages',
+                priority: 2,
+            },
         };
 
         var msgCodeParts  = code.split('.', 5);
         var principle     = msgCodeParts[1];
         var successCrit   = msgCodeParts[3].split('_').slice(0, 3).join('.');
         var techniques    = msgCodeParts[4].split(',');
         var techniquesStr = [];
+        function getPrefix(x) {
+            if (x.startsWith('ARIA')) {
+                return 'aria/';
+            }
+            if (x.startsWith('SCR')) {
+                return 'client-side-script/';
+            }
+            if (x.startsWith('C')) {
+                return 'css/';
+            }
+            if (x.startsWith('FLASH')) {
+                return 'flash/';
+            }
+            if (x.startsWith('F')) {
+                return 'failures/';
+            }
+            if (x.startsWith('G')) {
+                return 'general/';
+            }
+            if (x.startsWith('H')) {
+                return 'html/';
+            }
+            if (x.startsWith('PDF')) {
+                return 'pdf/';
+            }
+            if (x.startsWith('SVR')) {
+                return 'server-side-script/';
+            }
+            if (x.startsWith('SL')) {
+                return 'silverlight/';
+            }
+            if (x.startsWith('SM')) {
+                return 'smil/';
+            }
+            if (x.startsWith('T')) {
+                return 'text/';
+            }
+            return '';
+        }
```

Comment:
Unfortunately startsWith() isn't supported by IE 11. Suggest replacing it with the function used in the polyfill here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith (not the polyfill itself though)

Reply:
Will do